src/ServiceFabric/ServiceFabric/Commands/ServiceFabricClusterCertificateCmdlet.cs

```diff
@@ -230,26 +230,17 @@ protected virtual void Validate()
             }
         }
 
-        private void CreateSelfSignedCertificate(string subjectName, string keyVaultUrl, out string thumbprint, out CertificateBundle certificateBundle, out string outputFilePath)
+        private void CreateSelfSignedCertificate(string subjectName, string keyVaultUrl, out string thumbprint, out KeyVaultCertificateWithPolicy certificateBundle, out string outputFilePath)
         {
             outputFilePath = string.Empty;
-            var policy = new CertificatePolicy()
-            {
-                SecretProperties = new SecretProperties { ContentType = Constants.SecretContentType },
-                X509CertificateProperties = new X509CertificateProperties()
-                {
-                    Subject = subjectName,
-                    Ekus = new List<string> { "1.3.6.1.5.5.7.3.1", "1.3.6.1.5.5.7.3.2" }
```

Comment:
you are not adding this Ekus any more?

Reply:
I am not sure what would be the consequences of this but the cert might not work as expected